src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -44,6 +44,38 @@ def arbeitsl_geld_2_eink_m(  # noqa: PLR0913
     )
 
 
+def _arbeitsl_geld_2_eink_ohne_kindergeldübertrag_m(
```

Comment:
Somewhat confusing that `_arbeitsl_geld_2_eink_ohne_kindergeldübertrag_m` contains more components than `arbeitsl_geld_2_eink_m`

At some point we need to rename one of the functions and/or refactor the module.

Reply:
I agree, that's very confusing.

I renamed it in the following way:
`_arbeitsl_geld_2_eink_ohne_kindergeldübertrag_m` -> `arbeitsl_geld_2_eink_m` (=SGB 2 income without Kindergeldübertrag
`arbeitsl_geld_2_eink_m` -> `arbeitsl_geld_2_nettoeink_ohne_transfers_m` (Employment income, capital gains, etc.)

Do you think that's better?

Reply:
Sounds good!